README.md

```diff
@@ -54,6 +54,16 @@ To download all the sources, and place them in appropriate locations (i.e. $GOPA
 [installation script](bin/mainflux-install.sh). Once it completes, the script will provide the
 instructions on how to finish the manual installation (i.e. install the required infrastructure).
 
+Please note that the CoAP server is not installed with the installation script, but the CoAP server
```

Comment:
I think it would be better to add CoAP build to Makefile. CoAP should become a part of mainflux-1.0 release (even though this wil remain experimental).